<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>607</width>
    <height>527</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>20</y>
      <width>271</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Cộng Trừ Nhân Chia Phân Số</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBoxInput">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>80</y>
      <width>511</width>
      <height>161</height>
     </rect>
    </property>
    <property name="title">
     <string>Nhập tử số và mẫu số cho các phân số:</string>
    </property>
    <widget class="QLabel" name="labelTuSo1">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>141</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Tử số của phân số 1:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditTuSo1">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>30</y>
       <width>321</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="labelMauSo1">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>60</y>
       <width>141</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Mẫu số của phân số 1:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditMauSo1">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>60</y>
       <width>321</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditTuSo2">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>90</y>
       <width>321</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="labelTuSo2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>90</y>
       <width>141</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Tử số của phân số 2:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditMauSo2">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>120</y>
       <width>321</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="labelMauSo2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>120</y>
       <width>141</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Mẫu số của phân số 2:</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBoxChucNang">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>260</y>
      <width>511</width>
      <height>91</height>
     </rect>
    </property>
    <property name="title">
     <string>Chức năng:</string>
    </property>
    <widget class="QPushButton" name="pushButtonCong">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>71</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Cộng</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonTru">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>20</y>
       <width>71</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Trừ</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonNhan">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>20</y>
       <width>71</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Nhân</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonChia">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>20</y>
       <width>71</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Chia</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonTiep">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>20</y>
       <width>71</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Tiếp</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonThoat">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>20</y>
       <width>71</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Thoát</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBoxKetQua">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>360</y>
      <width>511</width>
      <height>91</height>
     </rect>
    </property>
    <property name="title">
     <string>Kết quả:</string>
    </property>
    <widget class="QLabel" name="labelKetQua">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>471</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background: rgb(255, 250, 235);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>607</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>lineEditTuSo1</tabstop>
  <tabstop>lineEditMauSo1</tabstop>
  <tabstop>lineEditTuSo2</tabstop>
  <tabstop>lineEditMauSo2</tabstop>
  <tabstop>pushButtonCong</tabstop>
  <tabstop>pushButtonTru</tabstop>
  <tabstop>pushButtonNhan</tabstop>
  <tabstop>pushButtonChia</tabstop>
  <tabstop>pushButtonTiep</tabstop>
  <tabstop>pushButtonThoat</tabstop>
  <tabstop>pushButton_7</tabstop>
  <tabstop>pushButton_8</tabstop>
  <tabstop>pushButton_5</tabstop>
  <tabstop>pushButton_6</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
